### ISSUE TITLE
Cloudflare is challenging RSS bots

### ISSUE THREAD

**Issue by kolardavid** [SteamDatabase/steamdb.info-issues#753]
- RSS reader
- Any
- All PatchnotesRSS pages

### Describe the bug

RSS buttons in patch notes leads to URL like https://steamdb.info/api/PatchnotesRSS/?appid=XXX , which shows RSS XML in the browser. Problem is, this URL is not in exclusion list for Cloudflare protection. This leads to a situation, that all RSS readers available fails to load this feed, because instead, they are served with HTML page informing about requirement of using newest browser and having JavaScript enabled.

Basically, while RSS is meant to be read by simple robots, whole SteamDB is protected by sophisticated filter requiring browser with real user behind it.

I would love to use RSS feeds to be notified about updates to my favorite games.

**Comment by xPaw** [comment]
Should be fine now.
